nextcord/polls.py

```diff
@@ -0,0 +1,441 @@
+    def from_dict(cls, data: DAPIPollAnswerPayload) -> Self:
+        return cls(answer_id=data["answer_id"], poll_media=PollMedia.from_dict(data["poll_media"]))
+
+
+class PollAnswerCount:
+    """The answer count for an answer in a poll.
+
+    .. versionadded:: 3.0
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The ID of the answer in question.
+    me_voted: :class:`bool`
+        Whether the bot has voted for this option.
+    count: :class:`int`
+        The number of users who have voted for this option.
+    poll: :class:`Poll`
+        The poll this answer count object represents.
+    """
+
+    __slots__ = ("id", "me_voted", "count", "poll")
+
+    def __init__(self, data: PollAnswerCountPayload, poll: Poll) -> None:
+        self.poll: Poll = poll
+        self.id: int = data["id"]
+        self.me_voted: bool = data["me_voted"]
+        self.count: int = data["count"]
+
+    def __repr__(self) -> str:
+        return (
+            "<PollAnswerCount "
+            f"id={self.id!r} "
+            f"me_voted={self.me_voted!r} "
+            f"count={self.count!r} "
+            ">"
+        )
+
+    def voters(
+        self, *, limit: Optional[int] = None, after: Optional[Snowflake] = None
+    ) -> AsyncIterator[Union[User, Member]]:
+        """|asynciter|
+
+        Returns an async iterator representing the users that have voted to this answer in the poll.
+
+        The ``after`` parameter must represent a member
+        and meet the :class:`abc.Snowflake` abc.
+
+        Examples
+        --------
+
+        Usage ::
+
+            # I do not actually recommend doing this.
+            async for user in count.voters():
+                await channel.send(f'{user} has voted for this answer!')
+
+        Parameters
+        ----------
+        limit: Optional[:class:`int`]
+            The maximum number of results to return.
+            If not provided, returns all the users who
+            voted for this answer.
+        after: Optional[:class:`abc.Snowflake`]
+            For pagination, votes are sorted by member.
+
+        Raises
+        ------
+        HTTPException
+            Getting the users for the votes failed.
+
+        Yields
+        ------
+        Union[:class:`User`, :class:`Member`]
+            The member (if retrievable) or the user that has voted to this answer.
+            The case where it can be a :class:`Member` is
+            in a guild message context. Sometimes it can be a :class:`User`
+            if the member has left the guild.
+        """
+
+        if limit is None:
+            limit = self.count
+
+        return answer_voters_iterator(self.poll.message, self.id, limit, after)
+
+    def to_dict(self) -> PollAnswerCountPayload:
+        payload: PollAnswerCountPayload = {
+            "id": self.id,
+            "me_voted": self.me_voted,
+            "count": self.count,
+        }
+        return payload
+
+
+class PollResults:
+    """The result of the poll.
```

Comment:
```suggestion
    """The result of a poll.
```
at least, or link to `Poll`